carbon-route-converter.html

```diff
@@ -20,18 +20,30 @@
 an app, please refer to [carbon-location](https://github.com/PolymerElements/carbon-route/blob/master/carbon-location.html)
 and [carbon-route](https://github.com/PolymerElements/carbon-route/blob/master/carbon-route.html).
 
-An example of a route element that describes
-`https://elements.polymer-project.org/elements/carbon-route-converter?foo=bar&baz=qux`:
+An example of a route object that describes
+`https://elements.polymer-project.org/elements/carbon-route-converter?foo=bar&baz=qux`
+and should be passed to other `carbon-route` elements:
 
     {
       prefix: '',
       path: '/elements/carbon-route-converter',
-      queryParams: {
+      __queryParams: {
         foo: 'bar',
         baz: 'qux'
       }
     }
 
+The private `__queryParams` property in the `route` object should be used only
```

Comment:
What about this wording instead? I'd like to include info on why we want other elements to be able to intermediate. I think it's also worthwhile to leave it open that there may be other routing elements, especially with an element like `carbon-route-converter`, where the intended user is a power user who's hacking around under the hood. 

`__queryParams` is private to discourage directly data-binding to it. This is so that routing elements like `carbon-route` can intermediate changes to the query params and choose whether to propagate them upstream or not. `carbon-route` for example will not propagate changes to its `queryParams` property if it is not currently active.